README.md

```diff
@@ -87,11 +87,20 @@ Accessibility should be incorporated and tested throughout the design and develo
 
 ## Policy & Informal
 
-* **Analytics:** All of our sites should have code snippets for 18F and Digital Analytics Program Google Analytics as well as Google Webmaster Tools.
 * Agency or Organization Communications Office
 * Legislative Affairs
 * Office of General Counsel
 
+### Analytics
+
+All of our sites should have code snippets for
+
+* 18F Google Analytics (18F only)
```

Comment:
Have we thought about under what circumstances it might make sense to only use the DAP, and not 18F's GA? A few thoughts:

* Shouldn't the program owner be the one to have the second GA account, if anyone? Could this lead to there being *3* analytics trackers in place, if the program owner also insists on their own non-DAP, non-18F profile?
* Is DAP access sufficient to get what we need from a site for analytics? What are the benefits of a separate account?

Right now, analytics.usa.gov itself is DAP-only, without an 18F-specific GA tracker. This is annoying, both because DAP moves slower and because I have to create Custom Reports to establish convenient defaults. But I'm also a GA novice, and maybe there's a way to create sub-views or something that replicate having a separate 18F GA account, without imposing on the DAP's limits or necessitating two trackers collecting identical information.